Source/SideMenuController.swift

```diff
@@ -190,6 +190,20 @@ open class SideMenuController: UIViewController, UIGestureRecognizerDelegate {
         super.viewWillDisappear(animated)
     }
     
+    // MARK: - Orientation -
+    
+    open override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return .portrait
+    }
+    
+    open override var preferredInterfaceOrientationForPresentation: UIInterfaceOrientation {
```

Comment:
I think we don't need a preferred orientation if we have only one supported orientation, but let's keep it. doesn't hurt anyone.